scripts/website-preview.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
So this kind of blends the two approaches we have been exposing in our other preview workflows, which is: (1) running the app directly via node and (2) running the app via docker.

With the docker workflow, we should not need to do the clone step. We should be relying on the published docker image, or in the case of the "local" workflow we can build the docker image locally from the remote repository without the clone step ([ref](https://stackoverflow.com/questions/26753030/how-to-build-docker-image-from-github-repository)). for consistency, this would all be instrumented in a make command (one for using the published image, one for building and using a local image).

For the node based workflow, we would do the clone step like you've got here then run the tf website with the correct env variables so it reads the local CDK content.

Reply:
https://github.com/hashicorp/terraform-cdk/pull/1652/commits/b50fa8658bc876d03deed96954fdb2579562f133 This eliminates the _clone_ step, and builds the Docker image from git

Reply:
https://github.com/hashicorp/terraform-cdk/pull/1652/commits/9a2f3aacb566287ceb8c5f58e7d36cacfc4159dc Thie replaces `npm run website-preview` with `make website-preview`